src/components/infoTooltips/ReserveFactorTooltip.tsx

```diff
@@ -0,0 +1,26 @@
+import { Trans } from '@lingui/macro';
+import { Link } from '../primitives/Link';
+
+import { TextWithTooltip, TextWithTooltipProps } from '../TextWithTooltip';
+
+interface ReserveFactorTooltipProps extends TextWithTooltipProps {
+  collectorLink?: string;
+}
+
+export const ReserveFactorTooltip = ({ collectorLink, ...rest }: ReserveFactorTooltipProps) => {
+  return (
+    <TextWithTooltip {...rest}>
+      <Trans>
+        Reserve factor is a percentage of interest which goes to a{' '}
+        {collectorLink ? (
+          <Link href={collectorLink} target="_blank">
```

Comment:
Please add `rel="noopener"` for any `target="_blank"` links as a best practice.  We'll want to practice this with new additions.  I recently added them to all existing new window links.  This attribute helps protect us because it prevents the website we link to from gaining access to our page/app via the window object.

Reply:
These are added automatically in our Link primitive: https://github.com/aave/interface/blob/92269df64a68a3502cb0fd8ff77f399e8503af0e/src/components/primitives/Link.tsx#L90, so my `target="_blank"` here is useless

Reply:
Right... I made that change, ha!

Reply:
Digging deeper on this topic, we can also support an `external` prop for `<Link/>`, as _we are assuming all uses of Link will be opening in a new tab_.  There's nothing wrong with that approach, but in cases where we don't want that to happen, we'd have the flexibility.  We could then add `target="_blank" rel="noopener"` only when the `external` prop is passed in.  Just some extra thoughts on this..

Reply:
Wow, disregard this last comment, should've read it first!

Reply:
The `Link` detects whether it's external based on the url (starting with 'http'). It's probably not as robust as an external prop, but good enough for most cases